src/makeRouteAndOptions.ts

```diff
@@ -27,6 +27,7 @@ export default function makeRouteAndOptions(
   const body = ((): string => {
     if (isObject(routeOrBody)) return JSON.stringify(routeOrBody)
     if (isObject(bodyAs2ndParam)) return JSON.stringify(bodyAs2ndParam)
+    if (bodyAs2ndParam instanceof FormData || bodyAs2ndParam instanceof URLSearchParams) return bodyAs2ndParam;
```

Comment:
don't we want to `JSON.stringify` this? Also, we need to add  `isBrowser` in front of this if since this would error on the server 😊

Reply:
a lot of cases to send FormData without JSON.stringify. Yes, will be done with isBrowser. Is it good?

Reply:
Add the `isBrowser` and we're good